src/js/modules/articles/components/summaries/RibbonComponent.js

```diff
@@ -1,18 +1,21 @@
 import React from "react";
-import { connect } from "react-redux";
 import { Link } from "react-router-dom";
 
-const RibbonComponent = ({ classes, article, section, featuredMedia }) => {
+const RibbonComponent = ({ classes, article }) => {
```

Comment:
Why RibbonComponent again? Pretty sure I asked this and you had a good reason, but I forgot lol